opendcs-rest-api-jetty/src/main/java/org/opendcs/odcsapi/jetty/Start.java

```diff
@@ -166,31 +176,42 @@ else if ("siteNameTypePreference".equalsIgnoreCase(n))
 		{
 			afr.read();
 		}
-		catch(Exception ex)
+		catch(Exception e)
 		{
-			String msg = String.format("Cannot read DB auth from file '%s': %s", afn, ex);
+			String msg = String.format("Cannot read DB auth from file '%s': %s", afn, e);
 			LOGGER.error(msg);
-			throw new StartException(String.format("Cannot read auth file: %s", ex));
 		}
 		ds.setUser(afr.getUsername());
 		ds.setPassword(afr.getPassword());
 		DbInterface.setDataSource(ds);
-		DbInterface.setDatabaseType(dbType);
+		try
+		{
+			DbInterface.setDatabaseType(dbType);
+		}
+		catch (Exception e)
```

Comment:
Do we want to continue if this throws an exception? Also if dbType is not null we are double logging